### ISSUE TITLE
standardisation of optical data

### ISSUE THREAD

**Issue by Geethen** [Clay-foundation/model#320]
I was wondering why the optical data requires standardisation and therefore the additional computation of the mean and std - An additional step that could be avoided if the optical data is radiometrically corrected. So for sentinel-2 divide by 10000 to get the digital values to reflectance which by default is in the range of 0-1.

Moreover, if the data is standardised, one usually needs to do a running mean/std computation over batches which if I understand correctly is an approximation of the actual mean/stdv. If I am not missing anything it makes more sense to use reflectance?

Second to this, if one had to fine-tune clay on reflectance data instead of standardised data would you foresee any issues?

**Comment by weiji14** [comment]
Yes, I agree that using reflectance would be better, and have thought about this at https://github.com/Clay-foundation/model/issues/94. I can reopen that thread, or continue the discussion here?

**Comment by srmsoumya** [comment]
@weiji14 I suggest we continue the discussion on the other thread.
